adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformFeatures.scala

```diff
@@ -22,32 +22,39 @@ import org.bdgenomics.adam.rdd.ADAMContext._
 import org.bdgenomics.utils.cli._
 import org.kohsuke.args4j.{ Argument, Option ⇒ Args4jOption }
 
-object Features2ADAM extends BDGCommandCompanion {
-  val commandName = "features2adam"
-  val commandDescription = "Convert a file with sequence features into corresponding ADAM format"
+object TransformFeatures extends BDGCommandCompanion {
+  val commandName = "transformFeatures"
+  val commandDescription = "Convert a file with sequence features into corresponding ADAM format and vice versa"
```

Comment:
If I have this correctly, on either `INPUT` or `OUTPUT` if the file extension is not recognized as a plain-text feature file, then it is assumed to be parquet.  That way this one command can convert both ways.  The `usage` strings below should be updated.